src/style.css

```
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+}
+
+div .stars__star:last-child {
+  margin: 0;
+}
```

1. it's better use class selectors for adding styles instead of tag ones
2. you can use :not(:last-child) instead of add margin at stars__star and reseting at :last-child

```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background-repeat: no-repeat;
  background-image: url(images/star.svg);
  background-position: center;
}

.stars__star:not(:last-child) {
  margin: 4px;
}
```